netci.groovy

```diff
@@ -221,34 +231,45 @@ def CreateStyleCheckTasks = { taskString, taskName, checkName ->
 // INNER LOOP TASKS
 // ----------------
 
-CreateBuildTasks('Windows_NT', null, null, "-winBlue", true, null, null)
+CreateBuildTasks(latestWindowsMachine, latestWindowsMachineTag, null, null, "-winBlue", true, null, null)
```

Comment:
I don't think we are supposed to be passing "-winBlue" for `testExtra` anymore. What does that flag even do? Enable WinBlue tests? Is the script so old that they weren't originally enabled by default?

Reply:
Note that this is passed in a bunch of different places and should be cleaned up. Maybe make the jenkins script pass the appropriate flag for wherever its running so we don't have to specify it here?

Reply:
I think -winBlue might enable "not Windows 7" tests which should probably continue running. Could be wrong but seems like we should just remove the winBlue tags and disable some tests on Windows 7.

Reply:
If I'm not mistaken there are significant platform differences across Win7 (Server 2008 R2) Win8 (Server 2012) WinBlue (Server 2012 R2), and Win10 -- and we have test platform configurations and test tags reflecting all of these. Worth an audit on the test tags as a separate work item, but whether or not we **currently** use this, our tests still "depend" on this on the appropriate platform.

However, I'll make sure that the "testExtra" accurately reflects that platform that each leg will actually run on. If modern = Win10 (Server 2016) then "-winBlue" is incorrect.